src/db/add_package.rs

```diff
@@ -130,6 +130,22 @@ pub(crate) fn add_package_into_database(
     Ok(release_id)
 }
 
+pub(crate) fn add_doc_coverage(
+    conn: &mut Client,
+    release_id: i32,
+    total_items: i32,
+    documented_items: i32,
+) -> Result<i32> {
+    debug!("Adding doc coverage into database");
+    let rows = conn.query(
+        "INSERT INTO doc_coverage (release_id, total_items, documented_items)
+            VALUES ($1, $2, $3)
```

Comment:
This should have an `ON CONFLICT UPDATE` for cases where we re-build the docs for an existing release.

Reply:
Ok!